spec/controllers/admin_outgoing_message_controller_spec.rb

```diff
@@ -55,6 +55,36 @@
 
     end
 
+    context 'unsuccessfully destroying the message' do
```

Comment:
I found that description slightly confusing - destruction occurred but that was undesirable? or unsuccessful attempt to destroy?